libnetwork/drivers/overlay/overlay.go

```diff
@@ -205,7 +188,7 @@ func (d *driver) initializeVxlanIdm() error {
 		return nil
 	}
 
-	d.vxlanIdm, err = idm.New(d.store, "vxlan-id", vxlanIDStart, vxlanIDEnd)
+	d.vxlanIdm, err = idm.New(nil, "vxlan-id", vxlanIDStart, vxlanIDEnd)
```

Comment:
Woohoo, datastore support can be ripped out of package `idm`, which is the only user of `bitseq.Handle`... WE CAN DELETE PACKAGE `bitseq` SOON! 🎉 🎉